documentation/manual/src/main/asciidoc/modules/infinispan.asciidoc

```diff
@@ -1014,87 +1020,92 @@ configure the caches for the most important data to have a synchronous CacheStor
 ====
 [source, XML]
 ----
-<infinispan><cache-container>
-  <distributed-cache-configuration name="configuration">
-     <locking striping="false" acquire-timeout="10000" concurrency-level="50" isolation="REPEATABLE_READ"/>
-     <transaction locking="PESSIMISTIC" mode="%s" /> # <1>
-     <expiration max-idle="-1" />
-     <indexing index="NONE" />
-     <state-transfer timeout="480000" await-initial-transfer="true" />
-  </distributed-cache-configuration>
-</cache-container></infinispan>"
+<infinispan>
+  <cache-container>
+    <distributed-cache-configuration name="configuration">
+      <locking striping="false" acquire-timeout="10000" concurrency-level="50" isolation="REPEATABLE_READ"/>
+      <transaction locking="PESSIMISTIC" mode="%s" /> # <1>
+      <expiration max-idle="-1" />
+      <indexing index="NONE" />
+      <state-transfer timeout="480000" await-initial-transfer="true" />
+    </distributed-cache-configuration>
+  </cache-container>
+</infinispan>
 ----
-<1> Transaction mode attribute will be filled with the value of the propery `hibernate.ogm.cache.transaction.mode`
+<1> Transaction mode attribute will be filled with the value of the property `hibernate.ogm.cache.transaction.mode`
 ====
 
-To apply a custom configuration to a newly created cache
-we need a previously defined cache or an existing __cache configuration__ on the remote Infinispan server to use as templates.
-
-There are two alternative ways to apply a cache configuration to new caches:
+If a different default configuration is defined on the Infinispan server, it's possible to use
+the property ``hibernate.ogm.datastore.cache_configuration`` to refer to it.
+This means that Hibernate OGM will create new caches using the selected configuration.
 
-* Using the dialect property: ``hibernate.ogm.datastore.cache_configuration``
-* Using the ``@CacheConfiguration`` annotation on Entity class
+If one some of your entities need a different configuration, you can use the annotation 
+``@CacheConfiguration``. This annotation will override the behavior of the global property
+and use the configuration defined for the annotated entity.
 
-The property is applied to all new caches, it means for all Entities. So property has a global scope.
-
-The annotations are applied to the new caches being used to store the respective Entities on which they are present,
-so the approach using annotations is scoped to the entity.
-
-It is possible to use a mixed approach; when both the dialect property and annotations are used the annotation takes
-precedence on the global property.
+Note that Hibernate OGM will create the caches only if they don't exist already and it doesn't
+update the configuration of existing caches.
 
 [IMPORTANT]
 ====
-Any configuration being referred to when when creating new Caches need to be present on the remote Infinispan cluster,
-or an exception will thrown.
+The cache configurations referred to by the annotation and the property must exist on
```

Comment:
what do you think of `the annotation or the property` ... `Otherwise an exception will be thrown`. ?

Reply:
I prefer the current wording but feel free to change it if you fell strong about it.
I replaced /will thrown/will be thrown/